migrations/db/init-scripts/00000000000000-initial-schema.sql

```diff
@@ -18,7 +18,15 @@ grant pg_read_all_data to supabase_read_only_user;
 create schema if not exists extensions;
 create extension if not exists "uuid-ossp"      with schema extensions;
 create extension if not exists pgcrypto         with schema extensions;
-create extension if not exists pgjwt            with schema extensions;
+do $$
+begin 
+    if exists (select 1 from pg_available_extensions where name = 'pgjwt') then
+        if not exists (select 1 from pg_extension where extname = 'pgjwt') then
+            create extension if not exists pgjwt with schema "extensions" cascade;
+        end if;
+    end if;
+end $$;
```

Comment:
Could we wait for merging https://github.com/supabase/postgres/pull/1489 before this one? I'm moving this `migrations/db/init-scripts/00000000000000-initial-schema.sql` file there.

Reply:
@steve-chavez sure. I am also trying to test and merge https://github.com/supabase/postgres/actions/runs/14040188774 into this one via rebase once i am done testing it too.

I'll focus on helping you get #1489 tests passing, and then we can rebase into this one hopefully by tomorrow, and restest with these changes (and test them on 17.4 version too)

Reply:
This was actually resolved by @steve-chavez in https://github.com/supabase/postgres/pull/1493 (thanks @steve-chavez !) once that is merged we'll rebase and get those changes into this branch, go through more testing and get this into staging

Reply:
Oh yes, and I think that #1493 needs to now be rebased into https://github.com/supabase/postgres/pull/1489 and then once #1489 is merged into develop, then we'll be good to go